### ISSUE TITLE
Add a cli option for "all domains my installer sees"

### ISSUE THREAD

**Issue by pde** [certbot/certbot#3629]
There should be a way to say, "the domains I want to select are all of the domains my installer plugin believes to be available".  This could look like `certbot --domains all`, or `certbot --all-domains`.

**Comment by pde** [comment]
Of course the local network ones would need to be filtered out, which we could get by calling `util.enforce_le_validity()`.

**Comment by bmw** [comment]
I'm assuming you created this ticket in response to potentially removing `dialog`. To me, it seems like a bit of a bad idea to run Certbot blindly without knowing what domains you're requesting. You could have missed domains, included domains you didn't want (see #3545), etc. What about instead we did something like:
```
Which names would you like to activate HTTPS for?
-------------------------------------------------------------------------------
1: example.org
2: www.example.org
-------------------------------------------------------------------------------
Select the appropriate numbers separated by commas and/or spaces or
simply press enter to select all above domains (Enter 'c' to cancel):
```
This makes it just as easy to select all domains when using the text interface.

**Comment by TheNavigat** [comment]
@bmw @pde This issue should probably be closed now since #3693 is merged.

**Comment by bmw** [comment]
Fixed in #3693.
